game/templates/partials/stortcuts_modal.html

```
@@ -0,0 +1,159 @@
+{% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)